src/custom/state/claim/hooks/index.ts

```diff
@@ -62,7 +62,9 @@ import useIsMounted from 'hooks/useIsMounted'
 import { ChainId } from '@uniswap/sdk'
 import { ClaimInfo } from 'state/claim/reducer'
 
-const CLAIMS_REPO_BRANCH = 'gip-13'
+// TODO: Revert this before it goes to production
+// const CLAIMS_REPO_BRANCH = 'gip-13'
```

Comment:
revert?

Reply:
Change back, that is what I meant

Reply:
No I mean like, change it back. In this pr